sdk/core/Azure.Core.TestFramework/README.md

```diff
@@ -220,7 +220,26 @@ In addition to the auto-rerecording functionality, using the RecordedTestAttribu
 
 ### Recording
 
-When tests are run in `Record` mode, session records are saved to the project directory automatically in a folder named 'SessionRecords'.
+When tests are run in `Record` mode, session records are saved automatically in a local folder named '.assets', located at the root of this repo. This folder will be created automatically by the Test Framework and should not be committed with other changes. Instead, recordings must be pushed manually to the [Azure SDK Assets](https://github.com/Azure/azure-sdk-assets) repo with the help of the `test-proxy` command line tool.
```

Comment:
Do we need to condition these docs on whether the migration has taken place yet?

Reply:
Since we intended to start moving recordings right after merging this PR, my original intent was to completely replace the docs with new ones. After today's discussion we're still not sure how things will be on the management side, so I'll take your suggestion and explain the two scenarios.

Reply:
Updated.